AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -756,16 +756,10 @@ describe('AddParticipantComponent', () => {
             const testCases = [
                 { searchJudiciaryEntriesValue: null, role: '', expectError: false },
                 { searchJudiciaryEntriesValue: null, role: 'Other', expectError: false },
-                { searchJudiciaryEntriesValue: null, role: 'Panel Member', expectError: true },
-                { searchJudiciaryEntriesValue: null, role: 'Winger', expectError: true },
```

Comment:
Actually could you re-add these but just change the expectations. We probably should look at another way to list specific test cases we want to test

Reply:
Done